Marlin/src/lcd/extui/ui_api.cpp

```diff
@@ -340,8 +340,10 @@ namespace ExtUI {
     #endif
   }
 
-  extruder_t getActiveTool() {
-    switch (active_extruder) {
+  extruder_t getTool(const uint8_t extruder) {
+    switch (extruder) {
+      case 7:  return E6;
+      case 6:  return E7;
```

Comment:
Seems like E6 and E7 got switched here?

Reply:
Thanks for noticing this glitch. Just fixed (cd59fa51000588338fc367c0c6f1d4c31b7da7ef).